src/core_io.h

```diff
@@ -37,7 +37,11 @@ bool DecodeHexBlockHeader(CBlockHeader&, const std::string& hex_header);
  */
 bool ParseHashStr(const std::string& strHex, uint256& result);
 std::vector<unsigned char> ParseHexUV(const UniValue& v, const std::string& strName);
-NODISCARD bool DecodePSBT(PartiallySignedTransaction& psbt, const std::string& base64_tx, std::string& error);
+
+//! Decode a base64ed PSBT into a PartiallySignedTransaction
+NODISCARD bool DecodeBase64PSBT(PartiallySignedTransaction& decoded_psbt, const std::string& base64_psbt, std::string& error);
+//! Decode a raw (binary blob) PSBT into a PartiallySignedTransaction
+NODISCARD bool DecodeRawPSBT(PartiallySignedTransaction& decoded_psbt, const std::string& raw_psbt, std::string& error);
```

Comment:
It seems a bit strange that `raw_psbt` is a string and not a `std::vector<unsigned char>`. The `DecodeBase64` call below also returns a `std::vector<unsigned char>`, so changing this would avoid a conversion (which I'm a bit surprised happens automatically).

Reply:
In fact, DecodeBase64 does not return a vector, it does return a std::string when given a std::string, which I imagine is why I ended up doing it this way, to avoid having to convert it? (When given a char*, it returns a vector. Your guess why it was done this way is as good as mine.)

I did carefully check that std::string is just a binary-safe as vector<char>. I'm fine leaving it or changing it as you prefer.

Reply:
Oh, you're right. That's weird, but probably out of scope for this PR. I'm fine with leaving this as is.